app/v2/template/get_template.py

```diff
@@ -0,0 +1,19 @@
+import decimal
+import datetime
+
+from flask import json, jsonify, request, url_for, current_app
+from werkzeug.exceptions import abort
+
+from app import api_user
+from app.dao import templates_dao
+from app.schema_validation import validate
+from app.v2.template import template_blueprint
+from app.v2.template.template_schemas import get_template_by_id_schema_response
+
+
+@template_blueprint.route("/<id>", methods=['GET'])
+def get_template_by_id(id):
+    template = templates_dao.dao_get_template_by_id(
+        id, request.args.get('version'))
```

Comment:
Ideally we want to avoid using a variable name that's a 'built-in', as it can be confusing. Could just replace with `template_id`?

Also would be nice to do some validation here, especially if this will be used as a public endpoint.